scripts/byoc-e2e-probe.py

```
@@ -0,0 +1,82 @@
+
+        req = ByocJobRequest(capability=CAP, payload={"prompt": "probe", "width": 512, "height": 512})
+        try:
+            submit_byoc_job(
+                req,
+                orch_url=ORCH,
+                signer_url=signer_url,
+                signer_headers=headers,
+                timeout=120.0,
+            )
+            print("submit_byoc_job: PASS")
+        except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)